nservicebus/endpoints/specify-endpoint-name.md

```diff
@@ -51,6 +55,13 @@ Specify a endpoint name when running the NServiceBus host: `/endpointName:"MyEnd
 
 NOTE: Only use code **OR** commandline/installation parameters.
 
+## When using NServiceBus.Hosting.Azure.HostProcess.exe
+
+### Defined in code
+
+Set the endpoint name using the `DefineEndpointName` extension method on the endpoint configuration.
```

Comment:
@danielmarbach original comment was about minor thing, rather than saying `DefineEndpointName` perhaps to use `.DefineEndpointName(endpointName)`. If you don't like it, discard my comment :)